nuxeo-core/nuxeo-core/src/main/java/org/nuxeo/ecm/core/trash/TrashService.java

```diff
@@ -288,6 +288,10 @@ default void untrashDocument(DocumentModel doc) {
      * @since 10.1
      */
     enum Feature {
+        /** Trashed state is deduced from lifeCycle. */
+        TRASHED_STATE_IS_DEDUCED_FROM_LIFECYCLE,
+        /** Trashed state is a dedicated property. */
```

Comment:
wrong copy/past comment, better to remove them all if they do not add anything

Reply:
I'll keep them to have some space between fields if it's ok ? I reworded the comment.